Src/VimCore/VimCore.fsproj

```diff
@@ -3,7 +3,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Vim.Core</RootNamespace>
     <AssemblyName>Vim.Core</AssemblyName>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>

```

Comment:
Will need to ifdef this change

Reply:
Should be able to just multi-target here between `net472` and `net6`